docs/automated-process.md

```diff
@@ -0,0 +1,30 @@
+# Automated Ansible Release Process
+
+## Preamble
+
+This document describes the (mostly) automated ansible community package release process.
+
+!!! note
+    Throughout this page, placeholder values in code blocks are formatted as
+    `${PLACEHOLDER_VALUE}` where `PLACEHOLDER_VALUE` describes the value to fill in.
+
+
+## Trigger the workflow
+
+1. Trigger [the automated workflow on GHA](https://github.com/ansible-community/ansible-build-data/actions/workflows/ansible-release.yml) and fill in the release version (like 11.2.0) and the major version (like 11).
+   The process will create a PR in the `ansible-build-data` repository, and wait for approval before continuing with uploading the package to PyPI.
```

Comment:
I believe this should be two sentences to avoid a comma splice.

Reply:
:+1: adffd2bdf9fcf79bb689070cb6fff2706cac8eb6.